apps/studio/src/store/modules/data/query/LocalQueryModule.ts

```diff
@@ -9,9 +9,35 @@ export const LocalQueryModule: DataStore<FavoriteQuery, DataState<FavoriteQuery>
     loading: false,
     error: null,
     pollError: null,
+    filter: undefined // maybe this can be more advanced? date filter?
   },
   mutations: mutationsFor<FavoriteQuery>({
     // more mutations go here
```

Comment:
So looks like this isn't implemented for queries and connections that come from the cloud?